worlds/messenger/Regions.py

```diff
@@ -6,16 +6,17 @@
     "The Shop": [],
     "Tower of Time": [],
     "Ninja Village": ["Candle", "Astral Seed"],
-    "Autumn Hills": ["Climbing Claws", "Key of Hope"],
+    "Autumn Hills": ["Climbing Claws", "Key of Hope", "Leaf Golem"],
     "Forlorn Temple": ["Demon King Crown"],
-    "Catacombs": ["Necro", "Ruxxtin's Amulet"],
+    "Catacombs": ["Necro", "Ruxxtin's Amulet", "Ruxxtin"],
     "Bamboo Creek": ["Claustro"],
-    "Howling Grotto": ["Wingsuit"],
-    "Quillshroom Marsh": ["Seashell"],
+    "Howling Grotto": ["Wingsuit", "Emerald Golem"],
+    "Quillshroom Marsh": ["Seashell", "Queen of Quills"],
     "Searing Crags": ["Rope Dart"],
     "Searing Crags Upper": ["Power Thistle", "Key of Strength", "Astral Tea Leaves"],
     "Glacial Peak": [],
-    "Cloud Ruins": ["Acro"],
+    "Cloud Ruins": [],
+    "Cloud Ruins Right": ["Acro"],
```

Comment:
are those logic fixes, or are those the additional requirements because of the added locations? if it's the latter, shouldn't that be on the location rather than the whole region?

Reply:
These are new locations that will always be created. This dict is what I use to create locations regardless of the user's settings.

Reply:
oh, i misinterpreted that. yeah, makes sense.